_includes/3d-secure-2.md

```diff
@@ -7,107 +7,118 @@
-        "cardholder": {
-            "firstName": "Olivia",
-            "lastName": "Nyhuus",
-            "email": "olivia.nyhuus@payex.com",
-            "msisdn": "+4798765432",
-            "homePhoneNumber": "+4787654321",
-            "workPhoneNumber": "+4776543210",
-            "shippingAddress": {
-                "addressee": "Olivia Nyhuus",
-                "email": "olivia.nyhuus@payex.com",
-                "msisdn": "+4798765432",
-                "streetAddress": "Saltnestoppen 43",
-                "coAddress": "",
-                "city": "Saltnes",
-                "zipCode": "1642",
-                "countryCode": "NO",
-            },{% else %}
-        "payer": {
+{% if api_resource == "creditcard" %}
+    "cardholder": {
+        "firstName": "Olivia",
+        "lastName": "Nyhuus",
+        "email": "olivia.nyhuus@payex.com",
+        "msisdn": "+4798765432",
+        "homePhoneNumber": "+4787654321",
+        "workPhoneNumber": "+4776543210",
+        "shippingAddress": {
+            "addressee": "Olivia Nyhuus",
             "email": "olivia.nyhuus@payex.com",
             "msisdn": "+4798765432",
-            "workPhoneNumber" : "+4787654321",
-            "homePhoneNumber" : "+4776543210",
-            "shippingAddress": {
-               "addressee": "Olivia Nyhuus",
-               "streetAddress": "Saltnestoppen 43",
-                "coAddress": "",
-                "city": "Saltnes",
-                "zipCode": "1642",
-                "countryCode": "NO"
+            "streetAddress": "Saltnestoppen 43",
+            "coAddress": "",
+            "city": "Saltnes",
+            "zipCode": "1642",
+            "countryCode": "NO",
+        },{% else %}
+    "payer": {
+        "email": "olivia.nyhuus@payex.com",
+        "msisdn": "+4798765432",
+        "workPhoneNumber" : "+4787654321",
+        "homePhoneNumber" : "+4776543210",
+        "shippingAddress": {
+            "addressee": "Olivia Nyhuus",
+            "streetAddress": "Saltnestoppen 43",
+            "coAddress": "",
+            "city": "Saltnes",
+            "zipCode": "1642",
+            "countryCode": "NO"
             }{% endif %}
-        },
-        "riskIndicator": {
-            "deliveryEmailAddress": "olivia.nyhuus@payex.com",
-            "deliveryTimeFrameIndicator": "01",
-            "preOrderDate": "19801231",
-            "preOrderPurchaseIndicator": "01",
-            "shipIndicator": "01",
-            "giftCardPurchase": false,
-            "reOrderPurchaseIndicator": "01"
-            "pickUpAddress" : {
-                "name" : "Company Megashop Sarpsborg",
-                "streetAddress" : "Hundskinnveien 92",
-                "coAddress" : "",
-                "city" : "Sarpsborg",
-                "zipCode" : "1711",
-                "countryCode" : "NO"
-            }
+}
+```
+
+{:.table .table-striped}
+| Field | Type | Description |
+| :---- | :--- | :---------- |{% if api_resource == "creditcard" %}
+
```

Comment:
This single linefeed is most likely destroying the entire table. Does it look OK in the browser?

```suggestion
```